bigtable/tests/unit/test_row.py

```diff
@@ -330,35 +335,56 @@ def test_delete_cells_with_string_columns(self):
                 column_qualifier=column2_bytes,
             ),
         )
-        self.assertEqual(row._pb_mutations, [expected_pb1, expected_pb2])
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[0],
+                         expected_pb1)
+        self.assertEqual(row.row_mutations.mutations_entry.mutations[1],
+                         expected_pb2)
```

Comment:
This misses the length check.

```python
        self.assertEqual(row.row_mutations.mutations_entry.mutations
                         [expected_pb1, expected_pb2])
```

Reply:
@tseaver I am getting error ```TypeError: Can only compare repeated composite fields against other repeated composite fields.```  I think it is because mutations is the protobuf message

Reply:
@aneepct: should we just create a MutationEntry for `expected_pb`?